lib/graphql/field/resolve.rb

```diff
@@ -51,7 +61,11 @@ def initialize(field)
         end
 
         def call(obj, args, ctx)
-          obj.public_send(@field.name)
+          if method_with_arguments?(@field.name, obj, args)
+            obj.public_send(@field.name, args)
```

Comment:
On specs (via http://graphql.org/learn/execution/#root-fields-resolvers) it says root resolvers may have context. Maybe I should pass the context to follow the specs, WDYT?